schema/testCase.xsd

```diff
@@ -194,6 +194,21 @@
           </xs:restriction>
         </xs:simpleType>
       </xs:attribute>
+      <xs:attribute name="isImplemented" use="required" default="TRUE">
```

Comment:
use="optional"

Reply:
No, because this works still with no attribute and forces the value. Optional gives us a "missing" third value to deal with. :)